azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/service/impl/AppServiceUtils.java

```diff
@@ -366,4 +372,17 @@ public static FunctionEntity fromFunctionAppEnvelope(FunctionEnvelope functionEn
                 .name((String) bindingProperties.get("name"))
                 .properties(bindingProperties).build();
     }
+
+    private static String getDockerImageNameFromLinuxFxVersion(String linuxFxVersion) {
+        String[] segments = linuxFxVersion.split(Pattern.quote("|"));
+        if (segments.length != 2) {
+            return null;
+        }
+        final String image = segments[1];
+        if (!image.contains("/")) {
+            return image;
+        }
+        segments = image.split(Pattern.quote("/"));
+        return segments[segments.length - 1].trim();
```

Comment:
what about image name like `user/image:version`?

Reply:
No need, because docker is not our core functionality